### PR TITLE
PATCH_FROM_DATA

### PR DESCRIPTION
**Pull request by prashantmhetreneo**
FormData Using patch method : RESTService.PATCH_FROM_DATA

### CODE REVIEW

File: lib/data/remote/request/rest_request.dart [hunk 1]
```diff
@@ -15,16 +15,16 @@ abstract class RESTRequest {
     this._apiKey = "";
   }
 
-  Future<Response> execute(String endpoint, Map<String, dynamic> params,
-      int apiCallMethod, int apiIdentifier,
+  Future<Response> execute(
+      String endpoint, dynamic params, int apiCallMethod, int apiIdentifier,
```
Comment (dhalloop7) [inline lib/data/remote/request/rest_request.dart:19]: No need of dynamic. Do it as it is

File: lib/data/remote/request/rest_request.dart [hunk 2]
```diff
@@ -15,16 +15,16 @@ abstract class RESTRequest {
...
       {forceRefresh: false}) async {
     return await _executeRESTCall(
         endpoint, -1, params, apiCallMethod, apiIdentifier,
         forceRefresh: forceRefresh);
   }
 
   Future<Response> _executeRESTCall(String endpoint, int resourceId,
-      Map<String, dynamic> params, int apiCallMethod, int apiIdentifier,
+      dynamic params, int apiCallMethod, int apiIdentifier,
```
Comment (dhalloop7) [inline lib/data/remote/request/rest_request.dart:27]: No need of dynamic params. Do it as it is

File: lib/data/remote/rest_service.dart [hunk 3]
```diff
@@ -218,7 +225,7 @@ class RESTService {
     return amerError;
   }
 
-  logParams(Map<String, dynamic> params) {
+  logParams(dynamic params) {
```
Comment (dhalloop7) [inline lib/data/remote/rest_service.dart]: No need of dynamic params. Do it as it is

**Review by dhalloop7 — CHANGES_REQUESTED** [review]
Please fix it

**Comment by prashantmhetreneo** [comment]
Ok, I will change dynamic to Map<String, dynamic>

**Pull request closed without merging**